lib/muffin_man/feeds/v20210630.rb

```diff
@@ -0,0 +1,44 @@
+module MuffinMan
+  module Feeds
+    require "json"
+    require "sp_api_helpers"
+    class V20210630 < SpApiClient
+
+      FEED_PATH = "/feeds/2021-06-30"
+
+      def create_feeds(feed_type, marketplace_ids, input_feed_document_id, params={})
+        @local_var_path = "#{FEED_PATH}/feeds"
+        @request_body = {"feedType": feed_type, "marketplaceIds": marketplace_ids, "inputFeedDocumentId": input_feed_document_id}.merge(sp_api_params(params))
+        @request_type = "POST"
+        call_api
+      end
+
+      def get_feeds(params)
+        @local_var_path = "#{FEED_PATH}/feeds"
+        sp_api_params = sp_api_params(params)
+        @query_params = sp_api_params.key?("nextToken") ?  sp_api_params.slice("nextToken") : sp_api_params
```

Comment:
Can we just pass the sp_api_params here and let the end user deal with nextToken logic? The reason I suggest this is that recently I came across an endpoint (I think it was searchCatalogItems maybe?) where passing ONLY the "nextToken" wouldn't work, you had to pass all of the params. If that were the case here it would lead to an incorrectly empty result.

Reply:
It will work both the ways like either user can pass `nextToken` separately or along with other parameter. In Doc they have mentioned to pass `nextToken` separately 
<img width="1275" alt="Screenshot 2022-10-17 at 12 54 12 PM" src="https://user-images.githubusercontent.com/110454114/196114284-4b3909ce-ee8e-4a0d-b2d6-609a3c875124.png">